### ISSUE TITLE
ArsMagica support

### ISSUE THREAD

**Issue by LemADEC** [WhiskyTangoFawks/WTFTweaks#6]
ArsMagica has a single block ID with both ores and blocks.
Notably, metadata 0 to 4 are Vintem ore (oreVinteum) up to Sunstone ore (oreSunstone),
while metadata 5 to 9 are the Moonstone block (blockMoonstone) up to Chimerite block (blockChimerite).
From what I can see, ore dictionary registration is correct on ArsMagica2 side, so I suppose WTFTweaks doesn't handle sub-types properly, hence causing mixing up blocks as ores?

**Comment by WhiskyTangoFawks** [comment]
Can you be specific as to what the problem actually is?  I'm not sure exactly what you're requesting.

**Comment by LemADEC** [comment]
You can't place Vinteum or Moonstone blocks because WTFTweak detects them as ores.
You need to place those blocks to invoke bosses in ArsMagica2.

**Comment by WhiskyTangoFawks** [comment]
You're right about the feature not being metadata specific.  I didn't bother making it metadata specific, because it's a bit strange for a mod to have ores and non-ores as the same block.  Doing a specific work around wouldn't be simple, especially as I don't use Ars Magica, and would involve recoding the feature to be metadata specific, or adding a whitelist feature.  As a work-around, you should be able to simple disable the feature temporarily with the config file, fight the boss, and turn it back on when you're ready to go back to mining.  Not ideal, I know, especially if you're trying to run a server, but better than nothing.  If I do another major release of WTFTweaks, I'll address this issue, but that may or may not happen.

**Comment by LemADEC** [comment]
I've proposed a more elegant solution in my other ticket: allow placing ores when no stone would be affected around it. I suspect this one is much easier to implement, adds a neat workaround for this issue and removes player frustation on block placement.

**Comment by LemADEC** [comment]
On a side note, it seems Github isn't updated against your latest binaries.
